scripts/module/debug.nut

```diff
@@ -9,10 +9,10 @@ function onPlayerCommand( player, cmd, text ) {
 
     collectgarbage();
     ReloadScripts();
-  }
-
-  if (cmd.tolower() == "pos") {
+  } else if (cmd.tolower() == "pos") {
     print(player.Pos);
+  } else if (cmd.tolower() == "help") {
```

Comment:
While you are at it, can you move the `.tolower()` like it's done in other scripts?